docs/source/references/schema.json

```
@@ -2392,7 +2392,39 @@
   "IpfsPath": "String",
   "Ipv4Addr": "Array<u8, 4>",
   "Ipv6Addr": "Array<u16, 8>",
-  "JsonString": "String",
+  "JsonString": {
+    "Enum": [
+      {
+        "tag": "Null",
+        "discriminant": 0
+      },
+      {
+        "tag": "String",
+        "discriminant": 1,
+        "type": "String"
+      },
+      {
+        "tag": "Bool",
+        "discriminant": 2,
+        "type": "bool"
+      },
+      {
+        "tag": "Array",
+        "discriminant": 3,
+        "type": "Vec<JsonString>"
+      },
+      {
+        "tag": "Number",
+        "discriminant": 4,
+        "type": "String"
+      },
+      {
+        "tag": "Object",
+        "discriminant": 5,
+        "type": "SortedMap<String, JsonString>"
+      }
+    ]
+  },
```

```suggestion
  "JsonValue: "JsonValue"
```

There was an idea that we can introduce a new `json` type into schema. It would be up to the sdk to know how to handle this type like it is for `Option<T>` or `Vec<T>`. What happened with this suggestion?

---

I'm not aware of this idea. 
I think the main difference is that Vec and Option has native representation in scale and json doesn't.